matchms/Scores.py

```diff
@@ -60,15 +64,20 @@ def __init__(self, references: ReferencesType, queries: QueriesType, similarity_
             List of query objects
         similarity_function
             Function which accepts a reference + query object and returns a score or tuple of scores
-
+        is_symmetric
+            Set to True when *references* and *queries* are identical (as for instance for an all-vs-all
+            comparison). By using the fact that score[i,j] = score[j,i] the calculation will be about
+            2x faster.
         """
-        Scores._validate_input_arguments(references, queries, similarity_function)
+        # pylint: disable=too-many-arguments
```

Comment:
Lint disable no longer needed
```suggestion
        # pylint: disable=too-many-arguments
```

Reply:
Still complained when I removed it.

Reply:
ok